examples/azure/terraform-azure-disk-example/main.tf

```diff
@@ -0,0 +1,42 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY AN AZURE MANAGED DISK
+# This is an example of how to deploy a managed disk.
+# ---------------------------------------------------------------------------------------------------------------------
+# See test/azure/terraform_azure_disk_example_test.go for how to write automated tests for this code.
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "azurerm" {
+  version = "~> 2.29"
+  features {}
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# PIN TERRAFORM VERSION TO >= 0.12
+# The examples have been upgraded to 0.12 syntax
```

Comment:
NIT: we have a newer comment block we now use as part of the [TF 0.13 testing/upgrade](https://github.com/gruntwork-io/terratest/pull/642) we're doing. Would you mind switching to that new one?